lib/Settings/Setting_wrapper.dart

```diff
@@ -0,0 +1,128 @@
+}
+
+class MyStatefulWidget extends StatefulWidget {
+  MyStatefulWidget({Key key}) : super(key: key);
+  @override
+  _MyStatefulWidget createState() => _MyStatefulWidget();
+}
+
+class _MyStatefulWidget extends State<MyStatefulWidget> {
+  final sections = ['People', 'Language', 'System', 'Help'];
+  final List<Map<String, List<Object>>> features = <Map<String, List<Object>>>[
+    {
+      'Guest': [Icons.account_circle]
+    },
+    {
+      'Language': [Icons.language, '/speech'],
+      'Speech': [Icons.record_voice_over, '/speech']
+    },
+    {
+      'Export': [Icons.cloud_upload, '/speech'],
+      'Import': [Icons.cloud_download, '/speech'],
+      'Display': [Icons.remove_red_eye, '/display'],
+      'Scanning': [Icons.center_focus_strong, '/speech'],
+      'Navigation': [Icons.chevron_right, '/navigation']
+    },
+    {
+      'User Help': [Icons.help, '/user help'],
+      'About Cboard': [Icons.info, '/about'],
+      'Donate': [Icons.monetization_on, '/speech'],
+      'Feedback': [Icons.feedback, '/speech']
+    },
+  ];
+  @override
+  Widget build(BuildContext context) {
+    return ListView.separated(
+      separatorBuilder: (BuildContext context, int index) {
+        return SizedBox(
+          height: 10,
+        );
+      },
+      itemCount: sections.length,
+      itemBuilder: (BuildContext context, int index) {
+        return Container(
+          padding: EdgeInsets.all(16.0),
+          decoration: new BoxDecoration(
+              color: Colors.white, borderRadius: BorderRadius.circular(5.0)),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Text(
+                sections[index],
+                textAlign: TextAlign.left,
+              ),
+              ListView.separated(
+                  separatorBuilder: (BuildContext context, int index) =>
+                      Divider(color: Color(0xffE0E0E0), indent: 78),
+                  physics: const NeverScrollableScrollPhysics(),
+                  shrinkWrap: true,
+                  itemCount: features[index].length,
+                  itemBuilder: (BuildContext context, int iconIndex) {
+                    return GestureDetector(
+                      onTap: index != 0
+                          ? () {
+                              Navigator.of(context).pushNamed(
+                                features[index].values.elementAt(iconIndex)[1],
+                              );
+                            }
+                          : null,
+                      child: ListTile(
+                        leading: Icon(
+                          features[index].values.elementAt(iconIndex)[0],
+                          color: Color(0xff391E75),
+                          size: 20,
+                        ),
+                        title: Text(features[index].keys.elementAt(iconIndex)),
+                        trailing: (index == 0)
+                            ? Container(
```

Comment:
When you pull from `master`, you can change this to a `Button` to link to a page that I made previously!

Reply:
I see there are  files relating to this `log-in` `sign-up` `welcome` which one should I link to ?

Reply:
Good question! Maybe ask design about this!